packages/roosterjs-content-model-dom/lib/modelApi/editing/applyTableFormat.ts

```diff
@@ -236,28 +240,36 @@ function formatCells(
     });
 }
 
-function setFirstColumnFormat(
+/**
+ * Set the first column format borders for the table
+ * @param rows The rows of the table
+ * @param format The table metadata format
+ */
+export function setFirstColumnFormatBorders(
     rows: ContentModelTableRow[],
-    format: Partial<TableMetadataFormat>,
-    metaOverrides: MetaOverrides
+    format: Partial<TableMetadataFormat>
 ) {
+    // Exit early hasFirstColumn is not set
+    if (!format.hasFirstColumn) {
+        return;
```

Comment:
Question, what if the table had first column before, then we want to use this function to remove the first column? If it just return here, will its border style remains no change?

Reply:
The function is only intended to set the style, not remove it. The borders will go back to normal after any call to applyTableFormat. There is however an issue with isHeader, so that if 'First Column' is set, then unset, the cells will remain as headers. So I think change should be made in formatCells to make isHeader false for every cell, and then reapply it if hasFirstColumn or hasHeaderRow is true.

Reply:
The issue is fixed on latest build.